src/tiled/mainwindow.ui

```diff
@@ -591,6 +592,14 @@
     <string>Edit Commands...</string>
    </property>
   </action>
+  <action name="actionZoom_With_Ctrl">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Zoom With Ctrl</string>
```

Comment:
I think "Mouse wheel zooms by default" is more descriptive (I cheated by looking up how they call this in Inkscape).